modules/kubernetes/ingress-healthz/templates/values.yaml.tpl

```diff
@@ -0,0 +1,13 @@
+application:
+  environment: ${environment}
+
```

Comment:
need a good way of setting the domain and certificate issuer.

Reply:
Yeah other then that I think it looks good.